app/services/monthly_statistics_timetable.rb

```diff
@@ -0,0 +1,69 @@
+module MonthlyStatisticsTimetable
+  # Currently, this uses the 3rd Monday of each month as agreed with Mike.
+  # It's not ideal, and may well change at a later date.
+  # It's also not cycle aware and will be needed to be updated yearly.
+
+  GENERATION_DATES = {
+    'October' => Date.new(RecruitmentCycle.previous_year, 10, 18),
```

Comment:
These and the dates below can almost certainly be automated

Reply:
:+1: